packages/core/package.json

```diff
@@ -6,8 +6,8 @@
     "storybook": "start-storybook -s .storybook/public -p 6006 --config-dir=.storybook",
     "dev:build": "build-storybook -s .storybook/public -c .storybook -o dist/storybook/core",
     "prelint": "npm link ../../scripts/eslint-plugin-clarity",
-    "lint": "eslint -c .eslintrc.js \"**/*.ts\" --ignore-pattern \"dist\" && stylelint --config .stylelintrc \"./**/*.scss\"",
-    "lint:fix": "eslint --fix -c .eslintrc.js \"./**/*.ts\" && stylelint --config .stylelintrc --fix \"./**/*.scss\"",
+    "lint": "eslint \"**/*.ts\" --ignore-pattern \"dist\" && stylelint \"./**/*.scss\"",
```

Comment:
@coryrylan, one small change after the review: removed path to config file for _eslint_ as well since default config file name is used.